src/jstree.js

```diff
@@ -93,6 +75,29 @@
 		idregex : /[\\:&!^|()\[\]<>@*'+~#";.,=\- \/${}%?`]/g,
 		root : '#'
 	};
+	
```

Comment:
The idea is really nice - I like it. I am just considering if it will be better to have a method for that, not a static function. That way any plugins can overwrite the default node - that could even mean faster redraws because of less work in the `redraw_node` function - what do you think?